### DISCUSSION TITLE
sc-glow-tts quality comparison with published version

### DISCUSSION THREAD

**Discussion by organic-chemistry** [coqui-ai/TTS#881] [General Q&A] [answered]
Hello, I synthethised the first sentence of the 
https://edresson.github.io/SC-GlowTTS/ website for the speaker p225.
by running 
`tts --text "Ask her to bring this things with her from the store" --model_name tts_models/en/vctk/sc-glow-tts  --speaker_idx p225 --out_path comp.wav `
Everything works fine however the quality is far from the one on the website with the SC-GlowTTS-Trans model trained either on 98 speakers or on 11 speakers.
Are the weight in the TTS package corresponding to the model trained on 98 speakers or 11 speakers.
Any idea why it is so different?
Best

**Comment by erogol** [comment, marked as answer]
We use a different phonemizer than the one used in the original work due to license conflict. It might be the reason.  But @Edresson  would know better.

**Reply by Edresson** [reply]
I think the main problem is the differences between phonemizers. Also in the demo I normalized the audios with the command: ` fmpeg-normalize "$wav_file" -nt rms -t=-24.63 -o "$wav_file" -ar 22050 -f`

**Reply by organic-chemistry** [reply]
Thank you for your answers.
So is it the quality of the new phonemizers that is a problem, or just the fact that it is not the one used for the training?
If it is the latter it courd be interesting to fine tune the model with the new phonemizer.

**Reply by erogol** [reply]
the latter. Yes, It'd be nice to fine-tune with it.
